CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added download progressbar in `util/net.py`
 ### Changed
 - Calls to hashlib now go through helper functions
+- Ported `scripts/docs.sh` to a CMD class in `dffml/service/dev.py`
```

Comment:
This let's users know what they should do as a result of the change

```suggestion
- Build docs using `dffml service dev docs`
```

Reply:
Okay , would keep this in mind for any future changes , thanks